browser/mapping.go

```diff
@@ -497,15 +504,22 @@ func mapPage(vu moduleVU, p *common.Page) mapping {
 			})
 		},
 		"goForward": p.GoForward,
-		"goto": func(url string, opts goja.Value) *goja.Promise {
+		"goto": func(url string, opts goja.Value) (*goja.Promise, error) {
+			gopts := common.NewFrameGotoOptions(
+				p.MainFrame().Referrer(),
+				p.MainFrame().NavigationTimeout(),
+			)
+			if err := gopts.Parse(vu.Context(), opts); err != nil {
+				return nil, fmt.Errorf("parsing frame navigation options to %q: %w", url, err)
```

Comment:
I forgot saying page here 🤦

```suggestion
				return nil, fmt.Errorf("parsing page navigation options to %q: %w", url, err)
```

Reply:
Or, just? for both?

```suggestion
				return nil, fmt.Errorf("parsing goto options to %q: %w", url, err)
```